tests/io/test_iceberg.py

```
@@ -0,0 +1,109 @@
+
+from cratedb_toolkit.cli import cli
+
+pl = pytest.importorskip("polars", reason="Skipping Iceberg tests because 'polars' package is not installed")
+
+if not hasattr(pd.DataFrame, "to_iceberg"):
+    pytest.skip("Older pandas releases do not support Apache Iceberg", allow_module_level=True)
+
+
+@pytest.fixture
+def example_iceberg(tmp_path) -> Path:
+    from pyiceberg.catalog import load_catalog  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)